driver/node.go

```diff
@@ -424,6 +440,15 @@ func (d *Driver) NodeExpandVolume(ctx context.Context, req *csi.NodeExpandVolume
 	})
 	log.Info("node expand volume called")
 
+	log.Infof("volume cap: +v\n", req.GetVolumeCapability())
```

Comment:
Was this for debugging? Shall we drop this?

Reply:
Sorry, looks like I didn't save in the editor before the last `git add` 🤦‍♂  I'll update shortly